Dockerfile

```diff
@@ -38,4 +38,4 @@ EXPOSE $PORT
 
 ENTRYPOINT ["tini", "--"]
 
-CMD ["yarn", "start"]
+CMD ["yarn", "start:debug"]
```

Comment:
This command leaks PII! Are we going to revert before we go to prod??

Reply:
We have a revert PR https://github.com/govuk-one-login/ipv-cri-bav-front/pull/197 for when perf testing is done 🚀